client/src/pages/searches/Search.js

```diff
@@ -1570,6 +1581,27 @@ const Search = ({
   function closeSaveModal() {
     setShowSaveSearch(false)
   }
+
+  function getResultsLimit() {
```

Comment:
This whole method could be replaced by:
```javascript
  function getResultsLimit() {
    return currentUser?.authorizationGroups
      ?.map(ag => ag.uuid)
      ?.includes(appSettings[UNLIMITED_EXPORTS_AUTHORIZATION_GROUP])
      ? 0
      : MAX_NR_OF_EXPORTS
  }
```
whic, since you're calling it only once, can then be inlined.